internal/network/manifests_generator.go

```diff
@@ -509,3 +511,66 @@ func NewConfig() (*Config, error) {
 	}
 	return &networkCfg, nil
 }
+
+const nodeIpHint = `
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: {{.ROLE}}
+  name: 10-{{.ROLE}}s-node-ip-hint
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,{{.NODE_IP_CONTENT}}
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/default/nodeip-configuration
+`
+
+// Add node ip hint (is supported from 4.11 but it makes no harm to push this file to any version)
```

Comment:
```suggestion
// Add node ip hint (is supported from 4.10 but it makes no harm to push this file to any version)
```

https://github.com/openshift/machine-config-operator/pull/3058 was merged